config_parser_test.cc

```diff
@@ -9,3 +12,55 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+TEST(NginxConfigTest, ToStringSimple) {
+	NginxConfigStatement statement;
+	statement.tokens_.push_back("foo");
+	statement.tokens_.push_back("bar");
+	EXPECT_EQ("foo bar;\n", statement.ToString(0)) << "foo bar wasn't serialized correctly";
+}
+
+class NginxStringConfigTest : public ::testing::Test {
+protected:
+	bool ParseString(const std::string& config_string) {
+		std::stringstream config_stream(config_string);
+			return parser_.Parse(&config_stream, &config_);
+
+	}
+	NginxConfigParser parser_;
+	NginxConfig config_;
+};
+
+
+TEST_F(NginxStringConfigTest, SimpleConfig) {
+	EXPECT_TRUE(ParseString("foo bar;")) << "Didn't parse successfully";
+	EXPECT_EQ(1, config_.statements_.size()) << "Didn't get one statement.";
+	EXPECT_EQ("foo", config_.statements_.at(0)->tokens_.at(0));
```

Comment:
This is good- but maybe use this same pattern on a more complicated example, to see if statements and tokens are still parsed correctly for a multi-line input or an input within braces.